themes/pom.xml

```diff
@@ -143,6 +143,13 @@
                     <include>**/node_modules/systemjs/dist/system.src.js</include>
                     <include>**/node_modules/tippy.js/dist/tippy.min.js</include>
                     <include>**/node_modules/upper-case/upper-case.js</include>
+                    <include>**/node_modules/parse-link-header/index.js</include>
+                    <include>**/node_modules/querystring/*.js</include>
```

Comment:
only in the new snowpack version right? but I could merge that in here as well

Reply:
It was my understanding that you added custom code so we could stop using querystring and url packages.  If that code is ready to go then let's go ahead and include with this PR.  Since it only affects My Resources, I think that will be easier than using a standalone PR for it.

Reply:
your understanding is correct, I thought that this would be included in your PR not this one. But we can add it here makes no difference.